src/lib/wdi5-control.ts

```diff
@@ -158,9 +175,16 @@ export class WDI5Control {
      * @param text
      */
     async enterText(text: string) {
+        let selector
```

Comment:
Nice, fluent async api for `enterText` 👍 . Could you comment in the test [here](https://github.com/ui5-community/wdi5/blob/7b7975e4f6097eb22906ea8a46c5090d73dbdd21/examples/ui5-js-app/webapp/test/e2e/error-logging.test.js#L154) for the fluent async api test? This would also close #294 .

Reply:
also done!